docs/static_site/src/pages/get_started/ubuntu_setup.md

```diff
@@ -234,13 +140,13 @@ If building on CPU and using MKL and MKL-DNN (make sure MKL is installed accordi
     ninja
 ```
 
-If building on GPU (make sure you have installed the [CUDA dependencies first](#cuda-dependencies)):
-Cuda 10.1 in Ubuntu 18.04 builds fine but is not currently tested in CI.
+For a GPU-enabled build make sure you have installed the
+[CUDA dependencies first](#cuda-dependencies)) and run:
 
 ```bash
     rm -rf build
     mkdir -p build && cd build
-    cmake -GNinja \
+    ~/.local/bin/cmake -GNinja \
```

Comment:
Is this path to cmake required?

Reply:
Yes, because we use pip install --user to install an up-to-date cmake. Often ~/.local/bin will be part of user's PATH. But it's not guaranteed and not true on default Ubuntu installation. So it's safer to specify the complete path.